doozer/tests/lockfile_prototype/test_generator.py

```
@@ -1263,6 +1266,141 @@ async def mock_resolve(config, image_pullspec=None):
+        repos = [
+            RepoEntry(
+                repoid="rhel-9-baseos-rpms",
+                baseurl="https://example.com/baseos/$basearch/os/",
+            )
+        ]
+
+        result = asyncio.run(
+            generator._resolve_stage_with_retry(
+                repo_list=repos,
+                arches=["x86_64", "aarch64"],
+                packages=["git", "dmidecode"] + failing_pkgs,
```

Ruff (0.15.20)

[warning] Consider `["git", "dmidecode", *failing_pkgs]` instead of concatenation

Replace with `["git", "dmidecode", *failing_pkgs]`

(RUF005)